common/components/AddressFieldFactory/AddressFieldFactory.tsx

```diff
@@ -12,6 +12,7 @@ interface DispatchProps {
 interface OwnProps {
   to: string | null;
   isSelfAddress?: boolean;
+  chainId: number;
```

Comment:
This component is already `connect`ed, it should get this from redux state instead of as a prop.

Reply:
I tried but I couldn't make this refactor work. Is it blocking?